openpype/hosts/houdini/api/lib.py

```diff
@@ -1054,3 +1054,71 @@ def add_self_publish_button(node):
     template = node.parmTemplateGroup()
     template.insertBefore((0,), button_parm)
     node.setParmTemplateGroup(template)
+
+
+def get_scene_viewer():
+    """
+    return an instance of a visible viewport.
+    There may be many, some could be closed, any visible are current
+    """
+    panes = hou.ui.paneTabs()
+    panes = [x for x in panes if x.type() == hou.paneTabType.SceneViewer]
+    panes = sorted(panes, key=lambda x: x.isCurrentTab())
+    if panes:
+        return panes[-1]
+
+    return None
+
+
+def sceneview_snapshot(
+        sceneview,
+        filepath="$HIP/thumbnails/$HIPNAME.$F4.jpg",
+        frame_start=None,
+        frame_end=None):
+    """
+    It takes snapshot of your scene view fot the given frame range.
+    So, it's capable of generating snapshots image sequence.
+    It works in different Houdini context e.g. Objects, Solaris
+
+    Example:
+        from openpype.hosts.houdini.api import lib
+        sceneview = hou.ui.paneTabOfType(hou.paneTabType.SceneViewer)
+        lib.sceneview_snapshot(sceneview)
+
+    Notes:
+        .png output will render poorly, so use .jpg.
+
+        How it works:
+            Get the current sceneviewer (may be more than one or hidden)
+            and screengrab the perspective viewport to a file in the
+            publish location to be picked up with the publish.
+
+        Credits:
+            https://www.sidefx.com/forum/topic/42808/?page=1#post-354796
+
+    Args:
+        sceneview (hou.SceneViewer): The scene view pane from which you want
+                                     to take a snapshot.
+        filepath (str): thumbnail filepath.
+        frame_start (int): the frame at which snapshot starts
+        frame_end (int): the frame at which snapshot ends
+    """
+
+    if frame_start is None:
+        frame_start = hou.frame()
+    if frame_end is None:
+        frame_end = frame_start
+
+    if not isinstance(sceneview, hou.SceneViewer):
+        log.debug("Wrong Input. {} is not of type hou.SceneViewer."
+                  .format(sceneview))
+        return
+    viewport = sceneview.curViewport()
+
+    # this will open an mplay window to show the result
```

Comment:
This comment is invalid because `outputToMPlay` is set to `False` below and thus this doesn't actually open an Mplay Window. Right?